New Entry and Terms: C++ Stack (#845)
Codecademy/docs: content/cpp/concepts/stacks/terms/push/push.md

---
Title: '.push()'
Description: 'Adds an element to the top of the stack.'
Subjects:
  - 'Computer Science'
  - 'Game Development'
Tags:
  - 'Objects'
  - 'OOP'
  - 'Classes'
  - 'Stacks'
CatalogContent:
  - 'learn-c-plus-plus'
  - 'paths/computer-science'
---

The **`.push()`** method adds an element to the top of the [`stack`](https://www.codecademy.com/resources/docs/cpp/stacks).

## Syntax

The `.push()` method is called on a `stack` using the following syntax:

```pseudo
stackName.push(value);
```

> The `value` must be of the same [data type](https://www.codecademy.com/resources/docs/cpp/data-types) that `stackName` was defined with.

Reviewer: ```suggestion
**Note:** Each `value` in `stackName` must be of the same [data type](https://www.codecademy.com/resources/docs/cpp/data-types).
```

## Codebyte Example

> The codebyte example below uses `.push()` to add several items to `myStack`:

Reviewer: ```suggestion
The following example uses adds several values to `myStack` with the `.push()` method:
```
